### PR TITLE
Feature/timer

### PR DESCRIPTION
**Pull request by mnuyens**
Fixed all the problems Andrew had with this and added more things to gitignore

### CODE REVIEW

File: types/Time.lcm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package forseti2;
+
+struct Time {
```
Comment (steakhouserodriguez) [inline types/Time.lcm:3]: This year we've done lower_case type defs, please make consistent

File: types/Time.lcm [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    header header;
+	int32_t game_time_so_far;
```
Comment (steakhouserodriguez) [inline types/Time.lcm:5]: probably makes sense to change to float

File: types/TimeControl.lcm [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package forseti2;
+
+struct TimeControl {
+    header header;
+	// either "pause", "start", or "reset"
+	string command_name;
```
Comment (steakhouserodriguez) [inline types/TimeControl.lcm:6]: protocol was designed this way last year due to the static udp wire protocol used to contact piemos
don't think there's an excuse to leave it this hacky this year

File: types/piemos_override.lcm [hunk 4]
```diff
@@ -0,0 +1,8 @@
+package forseti2;
+
+struct piemos_override
```
Comment (steakhouserodriguez) [inline types/piemos_override.lcm:3]: name is ambiguous

File: src/test_piemos_controller.py [hunk 5]
```diff
@@ -0,0 +1,25 @@
+import lcm
+import forseti2
+import time
+
+class TestPiemosController:
+
+    def __init__(self):
+        self.lc = lcm.LCM();
```
Comment (steakhouserodriguez) [inline src/test_piemos_controller.py:8]: please use url udpm://239.255.76.67:7667?ttl=1

File: src/piemos_controller.py [hunk 6]
```diff
@@ -0,0 +1,46 @@
+import lcm
+import forseti2
+import time
+
+class PiemosController:
+
+    def __init__(self):
+        self.lc = lcm.LCM()
```
Comment (steakhouserodriguez) [inline src/piemos_controller.py:8]: please use url udpm://239.255.76.67:7667?ttl=1

**Comment by steakhouserodriguez** [comment]
I mostly made style suggestions.
would like to revisit the game running architecture, but I realize this works. perhaps @chunjiang can advise.

**Comment by nikitakit** [comment]
The problems pointed out here should still be fixed. But I'm merging to consolidate the code into a single branch.

**Pull request merged**